health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -0,0 +1,701 @@
+        paginationTotalRows={state.totalRows}
+        onChangePage={handlePageChange}
+        onChangeRowsPerPage={handleRowsPerPageChange}
+        paginationRowsPerPageOptions={[5, 10, 15, 20]}
+      />
+      {showPopUp && (
+        <PopUp
+          className={"dataMapping"}
+          type={"default"}
+          heading={t("FACILITY_MAPPING_POP_HEADER")}
+          equalWidthButtons={true}
+          children={[
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)